rajawali/src/androidTest/java/org/rajawali3d/primities/ScreenQuadTest.java

```diff
@@ -0,0 +1,28 @@
+package org.rajawali3d;
+
+import static org.junit.Assert.*;
+import org.junit.*;
+
+import org.rajawali3d.primitives.ScreenQuad;
+
+/**
+ * @author Jared Woolston (jwoolston@keywcorp.com)
+ */
+public class ScreenQuadTest {
+    ScreenQuad screenQuad;
+
+    @Before
+    public void setup() throws Exception {
+        screenQuad = new ScreenQuad(false);
+    }
+
+    @After
+    public void teardown() throws Exception {
+        screenQuad = null;
+    }
+
+    @Test
+    public void testConstructor() throws Exception {
+        assertNotNull(screenQuad);
```

Comment:
Seems a little redundant. If a constructor fails it throws a checked exception otherwise the allocation happens therefore the setup function would fail for every test function.

Reply:
Working on a better test fixture, but I didn't want the test fixture work to hold up getting the ScreenQuad fix reviewed.